### DISCUSSION TITLE
[css] color variable을 왜 써야할까?

### DISCUSSION THREAD

**Discussion by al-bur** [woowacourse-study/2022-code-review-study-2#22] [General]
[color variable을 사용하라는 피드백 링크](https://github.com/woowacourse/javascript-youtube-classroom/pull/81#discussion_r825378185)

단순한 궁금증인데 왜 color variable을 사용하라고 하는 것일까요? 
사용해보신 분의 후기를 듣고싶습니다.!

**Comment by euijinkk** [comment]
1. 색상이 rgba나 #으로 표현되어 있는 것보다, 색상을 이름으로 표현하는 것이 유리하다.
2. 재사용 할 수 있다.
